tests/common/widgets/current-refinements/options.ts

```diff
@@ -24,9 +24,14 @@ export function createOptionsTests(
 ) {
   describe('options', () => {
     const searchClient = createAlgoliaSearchClient({});
+    const onSubmitListener = jest.fn();
 
     beforeEach(() => {
       searchClient.search.mockClear();
+
+      const form = document.createElement('form')!;
```

Comment:
sorry, that's not what I meant. The form can be created in the setup of the tests